src/util/write-payload.ts

```diff
@@ -106,6 +106,7 @@ export class WritePayload<T extends SpraypaintBase> {
             if (
               !this._isNewAndMarkedForDestruction(relatedModel) &&
               (idOnly ||
+                !this.model.isPersisted ||
                 this.model.hasDirtyRelation(key, relatedModel) ||
```

Comment:
I'd prefer to edit `hasDirtyRelation` instead so the logic is consistent (and no updates would need to be made here)

Reply:
How is this different than what was approved [here](https://github.com/graphiti-api/spraypaint.js/pull/47/commits/efe25fae419a1ad73d9dff2bd0f1ee6b81e932ed#diff-97b3d8324e220fbaab896cb31f347189L125)?

Reply:
I moved the test of isPersisted to `hasDirtyRelation` for both my change and that of [this commit](https://github.com/graphiti-api/spraypaint.js/pull/47/commits/efe25fae419a1ad73d9dff2bd0f1ee6b81e932ed#diff-97b3d8324e220fbaab896cb31f347189L125).